Swiftfin/Views/ItemEditorView/EditMetadataView/Components/Sections/LockMetadataSection.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Combine
+import JellyfinAPI
+import SwiftUI
+
+extension EditMetadataView {
+
+    struct LockMetadataSection: View {
+
+        @Binding
+        var item: BaseItemDto
+
+        var body: some View {
+            Section(L10n.lockedFields) {
+                Toggle(
+                    L10n.lockAllFields,
+                    isOn: $item.lockData.coalesce(false)
+                )
+            }
+
+            if item.lockData != true {
+                Section(L10n.active) {
```

Comment:
I personally think that negating what the web does and have `true/on` be "this is locked", which would kind of match "locking all fields". Also, we would be able to use this current implementation of the binding extensions.